app/processors/casino/processor_concern/service_tickets.rb

```diff
@@ -41,7 +41,12 @@ def clean_service_url(dirty_service)
 
         service_uri.path = (service_uri.path || '').gsub(/\/+\z/, '').sub('.html', '') # IE hack
         service_uri.path = '/' if service_uri.path.blank?
-        clean_service = service_uri.normalize.to_s
+
+        clean_service = begin
+                        service_uri.normalize.to_s
```

Comment:
does "normalize" raise the `InvalidURIError`?

Reply:
Specs do pass here. But they don't pass on master: https://travis-ci.com/github/konvenit/casino/builds/171597013. So I'd say everything works correctly.